indico/modules/events/papers/controllers/management.py

```diff
@@ -123,6 +125,23 @@ def _process(self):
                                               form=form)
 
 
+class RHManageContact(RHManagePapersBase):
+    """Allow to send emails to members of reviewing team"""
+
+    def _process(self):
+        paper_persons_dict = {}
+        for p in self.event.acl_entries:
+            paper_persons_dict[p]['judge'] = p.has_management_role('paper_judge', explicit=True)
```

Comment:
Yes, indeed I didn't test it. Don't know why I pushed it :worried:   

But if I do e.g. `paper_persons_dict[p] = {'judge': True}` and then `paper_persons_dict[p] = {'content_reviewer': True}` I will override the former. So perhaps it's better to keep using the `defaultdict` or am I missing something?